server/routes/index.js

```diff
@@ -34,10 +34,10 @@ router.use('/groups', groupsRouter.removeGroupRouter)
 //router.use('/groups', groupsRouter.removeMembersRouter)
 router.use('/groups', groupsRouter.addModeratorRouter)
 router.use('/groups', groupsRouter.removeModeratorRouter)
-//router.use('/groups', groupsRouter.updateGroupNameRouter)
-//router.use('/groups', groupsRouter.getGroupRouter)
-//router.use('/groups', groupsRouter.getAllGroupsRouter)
-//router.use('/groups', groupsRouter.getModeratorGroupsRouter)
+router.use('/groups', groupsRouter.updateGroupNameRouter)
+router.use('/groups', groupsRouter.getGroupRouter)
+router.use('/groups', groupsRouter.getAllGroupsRouter)
+router.use('/groups', groupsRouter.getModeratorGroupsRouter)
```

Comment:
This should do the job.
```suggestion
router.use('/groups', groupsRouter.updateGroupNameRouter)
router.use('/groups', groupsRouter.getModeratorGroupsRouter)
router.use('/groups', groupsRouter.getGroupRouter)
router.use('/groups', groupsRouter.getAllGroupsRouter)
```

Reply:
trying

Reply:
Worked!!